.takt/facets/instructions/validate-design.md

```
@@ -0,0 +1,32 @@
+技術設計ドキュメントの品質レビューを実施し、GO/NO-GO判定を行え。
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)